src/index.ts

```diff
@@ -86,14 +83,11 @@ function fetchAndLaunchMetals(context: ExtensionContext, javaHome: string) {
     ? serverVersionConfig
     : defaultServerVersion;
 
-  migrateStringSettingToArray("serverProperties");
-  migrateStringSettingToArray("customRepositories");
```

Comment:
these are not needed anymore, we're way past the migration

Reply:
Sounds good. To be honest, when I first put them in here I didn't know what they were for, and forgot to ask about them. So I'm glad they can go.

Reply:
We used to have these config as comma separated values and we eventually migrated to proper arrays.

The chance that some was using those options *and* did not upgrade metals yet (considering it's auto updated) is so low that I think is safe to drop them